pkg/network/usm/shared_libraries.go

```diff
@@ -135,7 +161,18 @@ func newSOWatcher(perfHandler *ddebpf.PerfHandler, rules ...soRule) *soWatcher {
 			byID:          make(map[pathIdentifier]*soRegistration),
 			byPID:         make(map[uint32]pathIdentifierSet),
 			blocklistByID: make(pathIdentifierSet),
+
+			libRegistered:               metricGroup.NewMetric("registered"),
+			libAlreadyRegistered:        metricGroup.NewMetric("already_registered"),
+			libBlocked:                  metricGroup.NewMetric("blocked"),
+			libUnregistered:             metricGroup.NewMetric("unregistered"),
+			libUnregisterNoCB:           metricGroup.NewMetric("unregister_no_callback"),
+			libUnregisterErrors:         metricGroup.NewMetric("unregister_errors"),
+			libUnregisterPathIDNotFound: metricGroup.NewMetric("unregister_path_id_not_found"),
 		},
+
+		libHits:    metricGroup.NewMetric("hits"),
+		libMatches: metricGroup.NewMetric("matches"),
```

Comment:
please add documentation about them as well

Reply:
the documentation is in the definition of soWatcher and soRegistery